mrp_production_project_estimated_cost/models/product.py

```diff
@@ -20,8 +20,8 @@
 import openerp.addons.decimal_precision as dp
 
 
-class ProductTemplate(models.Model):
-    _inherit = 'product.template'
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
 
     manual_standard_cost = fields.Float(
```

Comment:
Si quieres preguntamos a los clientes si se puede quitar, pero esto otra vez nos va a volver a hacer tocar los módulos de costes, el de Of ficticia y cualquier módulo que lo esté usando. 
Vuelvo a repetir que creo que no merece la pena ese esfuerzo ahora. Pero lo dicho. Si quieres envío un correo y pregunto